ios/Classes/codec/AblyFlutterReader.m

```diff
@@ -146,12 +146,10 @@ use an explicitly received null from Dart (manifesting this side as a nil id)
 
     [o addAgent:@"ably-flutter" version: FLUTTER_PACKAGE_PLUGIN_VERSION];
 
-    AblyFlutterClientOptions *const co = [AblyFlutterClientOptions new];
-    ON_VALUE(^(const id value) {
-        [co initWithClientOptions: o hasAuthCallback: value];
-    }, dictionary, TxClientOptions_hasAuthCallback);
-
-    return co;
+    
+    return  [[AblyFlutterClientOptions alloc]
+             initWithClientOptions:o
```

Comment:
can you change `o` for something more readable?

Reply:
Sure, that has been done in ed3851f23844f049f9285f462eb0096a7a5b845e. 😇